ecs-agent/api/ecs/interface.go

```diff
@@ -56,7 +56,7 @@ type ECSClient interface {
 	// and returns the endpoint at which this Agent should send system logs.
 	DiscoverSystemLogsEndpoint(containerInstanceArn string, availabilityZone string) (string, error)
 	// GetResourceTags retrieves the Tags associated with a certain resource
-	GetResourceTags(resourceArn string) ([]types.Tag, error)
+	GetResourceTags(ctx context.Context, resourceArn string) ([]types.Tag, error)
```

Comment:
I think we should document that this method accepts `containerName` context key which influences the HTTP headers.

Reply:
Updated